tools/ci_build/github/windows/setup_env_gpu.bat

```diff
@@ -1,2 +1,2 @@
-set PATH=C:\azcopy;C:\local\TensorRT-8.0.1.6.Windows10.x86_64.cuda-11.3.cudnn8.2\lib;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.1\bin;C:\local\cudnn-11.4-windows-x64-v8.2.2.26\cuda\bin;%PATH%
+set PATH=C:\azcopy;C:\local\TensorRT-8.0.3.4.Windows10.x86_64.cuda-11.3.cudnn8.2\lib;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.4\bin;C:\local\cudnn-11.4-windows-x64-v8.2.2.26\cuda\bin;%PATH%
```

Comment:
Please help remove cudnn dir from this file and setup_env_trt.bat. Because these files have been copied to CUDA installation dir.

Reply:
Okay, will do it later. Right now we want to let CI run and see whether it can pass all or not.